includes/class-capability.php

```diff
@@ -19,5 +30,34 @@ public function register() {
 				$role_obj->add_cap( self::MANAGE_REDIRECTS_CAPABILITY );
 			}
 		}
+
+		update_option( $capabilities_version_key, self::CAPABILITIES_VER );
+
+		return true;
+	}
+
+	/**
+	 * Unregister the capabilities
+	 *
+	 * @return void
+	 */
+	public function unregister() {
+
+		$capabilities_version_key = self::MANAGE_REDIRECTS_CAPABILITY . '_capability_version';
```

Comment:
This is the second time this line is used. Let's extract it into its own method for consistency.

Reply:
done